### ISSUE TITLE
Exploit docker support for labels on networks

### ISSUE THREAD

**Issue by dabooz** [open-horizon/anax#162]
Labels are a way to tag docker resources with metadata of our choosing. For example, this will allow us to explicitly identify which networks we have created in Anax, verses those that we do not try to control.

I have confirmed that after the upgrade of the go-dockerclient package, labels are available on the "create network" API and they do work, so now it's a matter of figuring out precisely how we want ot use them and implement it.

- [x] change current label prefix to openhorizon.anax (used to be network.bluehorizon.colonus)
- [x] implement and test the use of labels - in progress

**Comment by michaeldye** [comment]
That would be a nice feature. Right now we use labels to keep track of Docker containers that are managed by horizon; with networks we've been naming them after agreements. That convention is brittle and should be replaced by labels.

**Comment by dabooz** [comment]
I tried to make this move once before, but for the life of me I cannot remember what problem I ran into that caused me to give up. I think the problem was that docker supported labels but our go-dockerclient wasn't updated to support them. If that's still the case then we might want to ditch the go-docker client and create our own layer directly on the docker API. Or, we could extend/fork the go-dockerclient. That would probably take longer to do.
